lib/cheets.rb

```diff
@@ -0,0 +1,20 @@
+require './app'
+require 'data_mapper'
+
+DataMapper.setup(:default, 'postgres://localhost/chitter')
+
+class Cheets
+
+  include DataMapper::Resource
+  property :id,         Serial
+  property :body,       String
+  property :created_at, DateTime
+
+  DataMapper.finalize
```

Comment:
As discussed in the group session, this code probably belongs in a separate class referenced from your app.rb, or possibly in a rake file.